source/content/guides/composer-convert.md

```diff
@@ -1,156 +1,301 @@
 
-You can add these modules to your new codebase using Composer by running the following for each module in the `$site-composer` directory:
+1. When reviewing the site, take stock of exactly what versions of modules and themes you depend on. One way to do this is to change to run the `pm:projectinfo` Drush command from within a contributed modules folder (e.g. `/modules`, `/themes`, `/themes/contrib`, `/sites/all/themes`, `/sites/all/themes/contrib`, etc.).
 
-```{promptUser:user}
-composer require drupal/MODULE_NAME:^VERSION
-```
+  This command works on Drush 8. If you're using Drush 9, use `pm:list` or refer to [Drush Commands](https://drushcommands.com/drush-9x/pm/pm:projectinfo/):
 
-Where `MODULE_NAME` is the machine name of the module in question, and `VERSION` is the version of that module your site is currently using. Composer may pull in a newer version than what you specify, depending upon what versions are available. You can read more about the caret (`^`) [in the documentation for Composer](https://getcomposer.org/doc/articles/versions.md#caret-version-range-) itself.
+  ```bash{promptUser:user}
+  terminus drush $SITE.dev -- pm:projectinfo --fields=name,version --format=table
+  ```
 
-If you get the following error:
+  This will list each module followed by the version of that module that is installed.
 
-```none
-[InvalidArgumentException]
-Could not find a version of package drupal/MODULE_NAME matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
-```
+1. You can add these modules to your new codebase using Composer by running the following for each module in the `$SITE-composer` directory:
+
+  ```bash{promptUser:user}
+  composer require drupal/MODULE_NAME:^VERSION
+  ```
+
+  Where `MODULE_NAME` is the machine name of the module in question, and `VERSION` is the version of that module the site is currently using. Composer may pull in a newer version than what you specify, depending upon what versions are available. You can read more about the caret (`^`) in the [Composer documentation](https://getcomposer.org/doc/articles/versions.md#caret-version-range-).
+
+   - If you get the following error, the module listed in the error (or its dependencies) does not meet compatibility requirements:
+
+   ```none
+   [InvalidArgumentException]
+   Could not find a version of package drupal/MODULE_NAME matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
+   ```
+
+   If there is no stable version you can switch to, you may need to adjust the `minimum-stability` setting of `composer.json` to a more relaxed value, such as `beta`, `alpha`, or even `dev`. You can read more about `minimum-stability` in the [Composer documentation](https://getcomposer.org/doc/04-schema.md#minimum-stability).
```

Comment:
We should not recommend lowering the minimum stability to `dev`, as this can have odd side effects. We should instead tell people to require the specific version they need via Composer, e.g.
```
composer require drupal/ctools:^3.4@dev
```
If requiring a dev version fails because the dev version of the module requires a dev version of a dependency, they should fix this by also allowlisting the dev dependency at the same time:
```
composer require drupal/some-module:^1@dev org/some-dependency:^2@dev
```